src/string/immutable.zig

```diff
@@ -2003,6 +2023,7 @@ pub fn hasPrefixWithWordBoundary(input: []const u8, comptime prefix: []const u8)
     return false;
 }
 
+/// May fail with `error.OutOfMemory`.
```

Comment:
Instead of this, maybe just change the return type to `bun.OOM![]u8`, `Allocator.Error![]u8`, `error{OutOfMemory}![]u8`, or similar?